### DISCUSSION TITLE
Puzzler World cannot create a save file and won't load past error screen.

### DISCUSSION THREAD

**Discussion by bardemz** [rafaelvcaetano/melonDS-android#1192] [Support]
Hi all,

Pretty much the title.  I believe Puzzler World is not a DSiware game, so not sure what could be causing this issue.  I was able to replicate the problem on both a nightly from June (sorry, not sure which one exactly) and the Retroarch core, both on Android.  Curious if anyone knows of a fix.

I've tried deleting the .sav file that is created during the game's startup process, but still having no luck.

I hesitated to create an "issue" because I'm unsure if I'm doing something wrong.

Thanks!

![PXL_20230809_181608936](https://github.com/rafaelvcaetano/melonDS-android/assets/107502478/df8554fb-2064-4dc4-8177-9dfa43334307)

Edit to add: Puzzler World 2 (AKA Puzzler World 2011 in the EU) boots up just fine on the nightly build I'm using

**Comment by rafaelvcaetano** [comment]
Did you try it in the PC version of melonDS?

**Reply by bardemz** [reply]
Just downloaded 0.9.5 for PC and got the same error there as well:

![Screenshot 2023-08-09 180326](https://github.com/rafaelvcaetano/melonDS-android/assets/107502478/cfe30e57-989b-4999-a67b-819956aee3fa)

**Reply by rafaelvcaetano** [reply]
Phew. That means that I didn't mess anything in the port 😛 In that case, I recommend that you report this issue to them. They have much more knowledge on these kinds of issues than I have and will be able to provide a fix faster. You can report it on their repository: https://github.com/melonDS-emu/melonDS/issues

**Reply by bardemz** [reply]
Thanks for pointing me in the right direction, I was unaware these were separately maintained.

Just gotta say thanks for all your work; having a blast unlocking achievements on the standalone!
